continuous-improvement/choosing.md

```diff
@@ -97,7 +97,7 @@ title: Choosing what to improve
 
 <ul>
  <li>
-  <a href="./monitoring/GCTSS.html">
+  <a href="https://design.canada.ca/survey/index.html">
```

Comment:
@prycrane  What's our URL strategy again? Are we doing relative, absolute or the root in a global variable?

Reply:
I don't think we ever formalized it, but root with a global variable is probably better for long-term maintenance? 

<a href="{{ site.url }}/survey/">

Reply:
`<a href="{{ site.url }}/survey/"> `